electron-app/src/server/websites/ko-fi/ko-fi.service.ts

```diff
@@ -28,11 +24,13 @@ import { GenericAccountProp } from '../generic/generic-account-props.enum';
 import { LoginResponse } from '../interfaces/login-response.interface';
 import { ScalingOptions } from '../interfaces/scaling-options.interface';
 import { Website } from '../website.base';
+import _ = require('lodash');
 
 @Injectable()
 export class KoFi extends Website {
   readonly BASE_URL: string = 'https://ko-fi.com';
   readonly acceptsFiles: string[] = ['jpeg', 'jpg', 'png', 'gif'];
+  readonly acceptsAdditionalFiles: boolean = true;
```

Comment:
question: Was your branch behind? A lot of files are included in this that I think are just old PRs.

Reply:
Rebased; should be up-to-date and easier to compare now.